### PR TITLE
Create super-linter.yml

### PR DESCRIPTION
**Pull request by ShashwatAgrawal20**
This **PR** introduces the **linter** 

- Help establish coding best practices
- Preventing broken code from being pushed
-  Helping code reviews be a lot smoother

### CODE REVIEW

**Review by yactouat — APPROVED** [review]

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build --if-present
```
Comment (yactouat) [inline .github/workflows/node.js.yml:27]: hey @ShashwatAgrawal20 I dont think this one is necessary, as this project will be a lib that only contains logic and no build process

Comment (yactouat) [inline .github/workflows/node.js.yml:27]: hey @ShashwatAgrawal20 I dont think this one is necessary, as this project will be a lib that only contains logic and no build process

**Comment by yactouat** [comment]
@ShashwatAgrawal20 , in any case, this is great work ! ok for me to merge it ?

**Review by yactouat — APPROVED** [review]

**Comment by ShashwatAgrawal20** [comment]
@yactouat had a great experience working with you

**Pull request merged**